BD-3213 Add rules-based merging to Duplicate Users (#7608)
braze-inc/braze-docs: _docs/_user_guide/engagement_tools/segments/user_profiles/duplicate_users.md

## Rules-based merging

You can use rules to control how duplicate profiles are resolved when running a merge so the most relevent user profile is kept. When rules are set, Braze will keep profiles that match your criteria.

### Step 1: Define your rules

1. Go to **Audience** > **Manage Audience** > **Edit rules**.
2. In the **Profile to keep** section of the **Edit rules** panel, select the **Identifier** for the profiles that will be kept when merging duplicates. This can be the email address or phone number.
3. In the **Resolving ties** section, select the criteria that will determine how to solve ties between profiles with matching criteria from **Profile to keep**. You can select the following:<br>
- **Resolve ties using**: Created date, Updated date, Last session
- **Prioritization**: Newest, Oldest

![The "Edit rules" panel with sections to select options for "Profile to keep" and "Resolving ties".]({% image_buster /assets/img/audience_management/duplicate_users/edit_rules.png %})

> For example, you could keep the profile that has a phone number. If multiple users have the same phone number, you could resolve ties using the **Updated date** field and prioritze the newest user.

Reviewer: Updated date and newest would prioritize the most recently updated user.

> For example, you could keep the profile that has a phone number. If multiple users have the same phone number, you could resolve ties using the **Updated date** field and prioritze the newest user.

Author: ```suggestion
For example, you could keep the profile that has a phone number. If multiple users have the same phone number, you could resolve ties using the **Updated date** field and prioritize the most recently updated user.
```

{% endalert %}

![The "Manage Audience" page with "Merge all duplicates" highlighted.]({% image_buster /assets/img/audience_management/duplicate_users/bulk_merging/select_merge_profiles.png %}){: style="max-width:70%;"}

## Rules-based merging

You can use rules to control how duplicate profiles are resolved when running a merge so the most relevent user profile is kept. When rules are set, Braze will keep profiles that match your criteria.

### Step 1: Define your rules

1. Go to **Audience** > **Manage Audience** > **Edit rules**.

2. In the **Profile to keep** section of the **Edit rules** panel, select the **Identifier** for the profiles that will be kept when merging duplicates. This can be the email address or phone number.

3. In the **Resolving ties** section, select the criteria that will determine how to solve ties between profiles with matching criteria from **Profile to keep**. You can select the following:
- **Resolve ties using**: Created date, Updated date, Last session

> - **Prioritzation**: Newest, Oldest

Author: ```suggestion
- **Prioritization**: Newest, Oldest
```

![The "Edit rules" panel with sections to select options for "Profile to keep" and "Resolving ties".]({% image_buster /assets/img/audience_management/duplicate_users/edit_rules.png %})

For example, you could keep the profile that has a phone number. If multiple users have the same phone number, you could resolve ties using the **Updated date** field and prioritze the newest user.

### Step 2: Preview the results (optional)

After saving your rules, you can preview how they'll work by selecting **Generate a list of duplicates**. Braze will generate your preview and send it to your email address as a CSV file that shows which users would be kept and merged if your rules were applied. 

> ### Step 3: Merge duplices

Author: ```suggestion
### Step 3: Merge duplicates
```
